avrohugger-core/src/main/scala/matchers/TypeMatcher.scala

```diff
@@ -49,7 +49,14 @@ class TypeMatcher(
         case Schema.Type.NULL     => NullClass
         case Schema.Type.STRING   => StringClass
         case Schema.Type.FIXED    => sys.error("FIXED datatype not yet supported")
-        case Schema.Type.BYTES    => TYPE_ARRAY(ByteClass)
+        case Schema.Type.BYTES    =>
+          println(s"decimal: ${schema.toString(true)}")
```

Comment:
This can be removed, right?

Reply:
yes, I'm still debugging, waiting for @julianpeeters feedback to clean this up and move on to timestamps